aws/resource_aws_glue_crawler.go

```diff
@@ -254,15 +249,10 @@ func resourceAwsGlueCrawler() *schema.Resource {
 				},
 			},
 			"recrawl_policy": {
-				Type:     schema.TypeList,
-				Optional: true,
-				MaxItems: 1,
-				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
@ewbankkit do we want a semgrep rule for this?

Reply:
We have #7180 to track use of `suppressMissingOptionalConfigurationBlock`. I'll add a comment about a possible semgrep rule.
Thanks.